UNITTESTS/target_h/platform/cxxsupport/mstd_tuple

```diff
@@ -0,0 +1,93 @@
+/* mbed Microcontroller Library
```

Comment:
It is not possible to include the real handler ?

Reply:
I don't know, that's just how it's set up